modules/RTC/CodecSelection.js

```diff
@@ -2,12 +2,16 @@
 import { getLogger } from '@jitsi/logger';
 
 import * as JitsiConferenceEvents from '../../JitsiConferenceEvents';
-import { CodecMimeType, VideoCodecMimeTypes } from '../../service/RTC/CodecMimeType';
+import CodecMimeType from '../../service/RTC/CodecMimeType';
 import { MediaType } from '../../service/RTC/MediaType';
 import browser from '../browser';
 
 const logger = getLogger(__filename);
 
+// Default video codec preferences on mobile and desktop endpoints.
+const DESKTOP_VIDEO_CODEC_ORDER = [ CodecMimeType.VP9, CodecMimeType.VP8, CodecMimeType.H264 ];
+const MOBILE_VIDEO_CODEC_ORDER = [ CodecMimeType.VP8, CodecMimeType.VP9, CodecMimeType.H264 ];
```

Comment:
On P2P we want H.264 first though...

Reply:
Do we need to leave to out VP9 for p2p though ? If its at the end of the list, it will be negotiated but never used.

Reply:
I'd say that's ok